src/components/ListItem.js

```diff
@@ -74,6 +78,7 @@ const ListItem = ({ item, onDelete, token }) => {
       <button onClick={onDelete}>
         <img className="trash" src={trash} alt="delete item" />
       </button>
+      <Link to={'detail/' + item.name_normalized}>View Details</Link>
```

Comment:
```suggestion
      <Link to={`detail/${item.name_normalized}`}>View Details</Link>
```
Could use template literals here rather than concatenating strings, if you'd like. Both work just fine. 

Great! I liked @SaraSweetie 's suggestion. It is pretty standard to have an id vs a name in a route, however...I'm guessing because ids will be unique. Here, we have a requirement for no duplicate items, but I do think an id would be safer. Up to you.